content/help/ops/perf/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Performance Tuning
+description: Istio performance, sizing and tuning guide.
+weight: 90
+force_inline_toc: true
+---
+## Istio Performance Tuning
+
+Istio's goal is to have a performant out of the box configuration.
+However it is difficult to create a configuration that is performant for all use cases.
+
+The following section documents a few configuration knobs that affect performance.
+
+| Knob | Default | Symptom | Description |
+| --- | --- | --- | --- |
+| [`Sidecar resource`](/docs/reference/config/networking/v1alpha3/sidecar) | Disabled | High memory usage by sidecars and high CPU by Pilot | Enable namespace isolation to get a significant reduction in sidecar memory usage and pilot CPU usage. |
+| `global.proxy.concurrency` | `2` | Sidecar resource utilization and latency.| This values should be at least equal to the number of CPU core allocated to the proxy. |
+| `pilot.keepaliveMaxServerConnectionAge` | `30m` | Uneven load distribution across Pilot replicas. | Reduce this number if you want quicker balancing of load at the cost of increased system churn. |
```

Comment:
The "symptom" column ca
```suggestion
| `pilot.keepaliveMaxServerConnectionAge` | `30m` | A long time can cause uneven load distribution across Pilot replicas. | Reduce this number if you want quicker balancing of load at the cost of increased system churn. |
```

Reply:
The uneven distribution may happen anyway, but if a shorter keepalive age is used, this situation will be fixed faster. What about 'Uneven load distribution across Pilot replicas for a longer period of time'?